src/EnergyPlus/IOFiles.hh

```diff
@@ -380,6 +381,12 @@ public:
                 // The Fortran 'G' format insists on a leading 0, even though
                 // that actually means losing data
                 specs_.type = 'Z';
+
+                // We'll be formating a number between -1 and 1 as scientific notation, so we want some amount of decimals to be meaningful
+                if (specs_.precision == -1) {
+                    specs_.precision = 4;
+                }
```

Comment:
In the N case, default to precision = 4.

Reply:
I suppose if we were fine with having around 6 precision, we could delete that block and let fmt default it in the Z case which it forwards to.

Reply:
Yeah I wouldn't care either way, but I think it's reasonable, and we don't need to belabor this corner case.

Reply:
Is this the default precision for the Fortran output? This whole code is to emulate Fortran output styles within the fmt library.